### DISCUSSION TITLE
Présence par semaine

### DISCUSSION THREAD

**Discussion by mxtla** [danymat/INSAnonym-utils#75] [Q&A, Bug report & Support]
Pour le fichier d'attaque, si un utilisateur n'apparait pas dans le fichier de base pour une semaine donnée, pour le fichier json, faut il spécifier quelque chose (comme "DEL"), ou laisser le champ vide ?

Aussi, quel est l'impact sur le score si il n'y avait pas de trace pour une semaine donnée d'un identifiant mais que l'on propose quand même quelque chose ? En particulier, est-ce que on sera pénaliser dans le score et qu'il aurait mieux valu que l'on mette "DEL" ?

Merci

**Comment by mxtla** [comment]
Et combien de semaines doit-on considérer ? de 0 à 11 ? 

Merci

**Comment by danymat** [comment]
Bonjour, 
- Si un utilisateur n'apparait pas dans le fichier d'attaque, il faut spécifier null, comme ceci:
```json
...
 "79":{
    "2015-14":null,
    "2015-15":null,
    "2015-16":null,
}
...
```
Dans ces cas, cela veut dire que selon vous, l'id 79 correspond à pas d'identifiant a la semaine 14, 15 et 16.

Si vous mettez `DEL` à la place de `null`, cela veut dire que vous pensez que l'identifiant à été supprimé dans le fichier anonymisé à la semaine donnée.

**Reply by danymat** [reply]
D'ailleurs je viens de voir dans la doc utilisateur, il est spécifié:

- If a user is not present in the given week in original file (e.g the id 7 is not present at week 12), the server will automatically skip this week for the id (No score will be given for this user at the given week.)
